openstudio-common-measures.gemspec

```diff
@@ -26,11 +26,19 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '~> 2.7.0'
+  spec.required_ruby_version = '~> 3.2.2'
 
-  spec.add_dependency 'bundler', '>= 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.7.0'
-  spec.add_dependency 'openstudio-standards', '~> 0.5.0' 
+  spec.add_dependency 'bundler', '~> 2.4.10'
```

Comment:
Can you make bundler less restrictive `'bundler', '~> 2.4.10'`? The same goes for any other dependency, it would be best to be as open as possible.